src/components/CheckoutForm.js

```diff
@@ -6,38 +6,49 @@ import {
   injectStripe
 } from "react-stripe-elements";
 import axios from "axios";
+import { connect } from "react-redux";
 import { Form, Button, Card, Container } from "semantic-ui-react";
 
 class CheckoutForm extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      success: "",
-      error: "",
       renderStripeForm: true
     };
   }
 
-  submitPayment = async () => {
+  submitPayment = async ev => {
+    ev.preventDefault();
+    await this.props.stripe.createToken().then(({ token }) => {
+      if (token) {
```

Comment:
I believe this 'if' 'else' statement should have the ternary function syntax.

Reply:
My I ask why? I mean, we could go with either one, but we've been using if else statements throughout the entire app, and to maintain consistency I think we should stick with it.

Reply:
I have been asked to do this on previous PRs because it has notably less code.  The example I had to convert to a ternary is in lines 35 and 37 from
https://github.com/CraftAcademy/gnosis-client/blob/development/src/components/DisplayArticles.js

Reply:
Fixed!